pymysql/connections.py

```diff
@@ -641,6 +633,28 @@ def _config(key, arg):
             port = int(_config("port", port))
             bind_address = _config("bind-address", bind_address)
             charset = _config("default-character-set", charset)
+            if not ssl:
+                ssl = {}
+            ssl_config_params = {
+                "ca": _config("ssl-ca", ssl.get("ca")),
+                "capath": _config("ssl-capath", ssl.get("capath")),
+                "cert": _config("ssl-cert", ssl.get("cert")),
+                "key": _config("ssl-key", ssl.get("key")),
+                "cipher": _config("ssl-cipher", ssl.get("cipher")),
+            }
+            # Update parameters, but parameters coming from the class parameter
+            # must have preference.
+            for key, value in ssl_config_params.items():
+                if value:
+                    ssl.setdefault(key, value)
```

Comment:
Using    ssl[key] = value   would overwrite the value that's already there, and this gives priority to the values that are set via the ssl dict.